man/gcolor3.1

```diff
@@ -0,0 +1,78 @@
+.TH GCOLOR3 "1" "August 26, 2018"
+.
+.SH NAME
+gcolor3 \- color selection dialog
+.
+.SH SYNOPSIS
+.B gcolor3
+.RI [ OPTIONS ]
+.
+.SH DESCRIPTION
+.B gcolor3
+enables you to pick the color from any pixel on your screen. It also offers a palette, so that you can easily mix and match a couple of colors together.
+.
+.SH OPTIONS
+.SS "Help Options"
+.TP
+.BR \-h ", " \-\^\-help
+Show help options
+.TP
+.BR \-\^\-help\-all
+Show all help options
+.TP
+.BR \-\^\-help\-gtk
+Show GTK+ options
+.SS "GTK+ Options"
```

Comment:
I viewed some man pages of other GNOME applications, and none included these. Can you please remove them?

Reply:
Only included them because --help did, but no problem!